Makefile

```
@@ -383,3 +383,11 @@ documentation: check-unused-components generate-public-docs verify-api-docs
 update-redocly-cli:
 	rm -f package-lock.json
 	npm install @redocly/cli@latest
+
+# Local image build (same Containerfile and BUILD_DATE arg as .github/workflows/ci.yml docker-build-push; CI also sets multi-arch push and registry tags).
+DOCKER_IMAGE_LOCAL ?= eval-hub:local
+DOCKER_BUILD_DATE ?= $(shell date -u +%Y-%m-%dT%H:%M:%SZ)
+DOCKER ?= podman # or docker
```

checkmake (0.3.2)

[warning] Variable "DOCKER_BUILD_DATE" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)